x-pack/metricbeat/module/iis/application_pool/_meta/data.json

```diff
@@ -9,7 +9,28 @@
         "application_pool": {
             "name": "test.local",
             "net_clr": {
-                "total_exceptions_thrown": 0
+                "total_exceptions_thrown": 0,
```

Comment:
Since we don't have the ASP.NET application setup to test the real metrics flow. This step will be done later by identifying a mechanism to test the actual metrics generated by running the .NET application.